2-mandatory/3-shopping-list.js

```diff
@@ -19,8 +19,23 @@ let pantry = {
 };
 
 function createShoppingList(recipe) {
-  // write code here
-}
```

Comment:
The implementation looks good to me. For an extra challenge - can you implement this using the `filter` array method?